### PR TITLE
Doc upload optimizations

### PR DESCRIPTION
**Pull request by rjohnson2011**
## Description of change
Introduces the following changes in an effort to reduce file upload time:

-   Add logging of file size so we can correlate file upload times with file size.
 -  Change EVSS::BaseService to use request argument for setting timeouts.
 -  Increase minimum retry wait time for DocumentUpload jobs to 1 hour.
-   Safely time out the upload after 8 minutes.

## Original issue(s)
[Investigate EVSS DocumentUpload Job#47991](https://app.zenhub.com/workspaces/platform-tech-team-3-6335ab9b1901b99243ce7601/issues/department-of-veterans-affairs/va.gov-team/47991)

### CODE REVIEW

**Review by RachalCassity — APPROVED** [review]

File: app/workers/evss/document_upload.rb [hunk 1]
```diff
@@ -1,12 +1,18 @@
 # frozen_string_literal: true
 
 require 'ddtrace'
+require 'evss/base_service'
```
[the remark this answers is not in the document]
Comment (rjohnson2011) [reply app/workers/evss/document_upload.rb:4]: Nope, relic I meant to take out, nice catch!

File: lib/evss/base_service.rb [hunk 2]
```diff
@@ -44,18 +44,15 @@ def base_url
     end
 
     def timeout
-      self.class::DEFAULT_TIMEOUT
+      { timeout: self.class::DEFAULT_TIMEOUT }
```
[the remark this answers is not in the document]
Comment (rjohnson2011) [reply lib/evss/base_service.rb]: Sounds good, I'll add those timeouts and set to the default for the time being.

File: lib/evss/base_service.rb [hunk 3]
```diff
@@ -44,18 +44,17 @@ def base_url
     end
 
     def timeout
-      self.class::DEFAULT_TIMEOUT
+      { timeout: self.class::DEFAULT_TIMEOUT,
+        open_timeout: self.class::DEFAULT_TIMEOUT,
+        write_timeout: self.class::DEFAULT_TIMEOUT }
```
Comment (ericboehs) [inline lib/evss/base_service.rb:49]:
```suggestion
      { 
        timeout: self.class::DEFAULT_TIMEOUT,
        open_timeout: self.class::DEFAULT_TIMEOUT,
        write_timeout: self.class::DEFAULT_TIMEOUT
      }
```

Normally I'd write it like this. But Rubocop may allow this /shrug

**Review by ericboehs — APPROVED** [review]

**Review by ericboehs — DISMISSED** [review]

File: lib/evss/base_service.rb (hunk 3 above)
Comment (rjohnson2011) [reply lib/evss/base_service.rb]: Toats, was in between doing it like this.

**Review by ericboehs — APPROVED** [review]

**Pull request merged**